### ISSUE TITLE
Oculus OpenXR: unable to recover if headset is taken off or moved from face during play session

### ISSUE THREAD

**Issue by Nocxr** [GodotVR/godot_openxr#28]
Using the Oculus OpenXR runtime If the headset is removed or moved away from the face/head enough that the sensor turns the screen off Godot throws these errors and will not recover until play is restarted:
```
ERROR: OpenXRApi::process_openxr: Abort Mission!
   At: src\OpenXRApi.cpp:1508
ERROR: OpenXRApi::xr_result: OpenXR failed to wait for swapchain image! [XR_ERROR_CALL_ORDER_INVALID]
```
It will continue to spam the XR_ERROR_CALL_ORDER_INVALID line regardless of placing the headset back on, and the HMD screen will rapidly flash blank gray until the play session is ended or restarted.  This is not a huge deal as the initial play session works fine but if your trying to lift the headset up/off to check something on the screen it makes things a little more difficult.

Info:
Windows 10 x64 20H2 19042.746
Radeon Driver: 21.1.1
Godot Version: Godot_v3.2.4-beta6_win64
OpenXR Loader: 1.0.12
Oculus Quest 1 using Oculus Link
Quest Firmware: 1201830.9360.0
Oculus App Version 23.0.0.43.517 (23.0.0.43.517)

**Comment by HyeokSuLee** [comment]
This still problem. 
When i taken off hmd and back, display only shows black screen.
I think need some method to recover session after "OpenXR EVENT: session state changed to XR_SESSION_STATE_STOPPING"

**Comment by BastiaanOlij** [comment]
I could not reproduce this with commit bb85fd21429733d9d4d76319463717f78f7ecdfe on Windows (which bring OpenXR up to 1.0.16). Not sure if that is because some of the changes were already merged before or if this is because I'm using a Rift S on Windows. This may be an issue with the Quest.

**Comment by HyeokSuLee** [comment]
Hi, thanks for your effort.
I'm using lift S too.
So I'v tried with "newest build module ( libgodot_openxr.dll )" + "loader ( openxr_loader.dll -1.0.16 )" but it doesn't show anything in hmd.
Well, "newest build module" + "loader(1.0.12)" works good, but same issue above.

Can i get your loader file? (1.0.16)

**Comment by BenMcLean** [comment]
I have been thinking I should wait to switch my project over to OpenXR until this issue is fixed, but now I'm thinking maybe instead I should switch now to provide more feedback? Or is that needed at this point?

**Comment by HyeokSuLee** [comment]
> I have been thinking I should wait to switch my project over to OpenXR until this issue is fixed, but now I'm thinking maybe instead I should switch now to provide more feedback? Or is that needed at this point?

use this instead https://github.com/GodotVR/godot_openvr. It works fine.

**Comment by BenMcLean** [comment]
> use this instead https://github.com/GodotVR/godot_openvr. It works fine.

That's what I've been using so far but I'd like to switch to OpenXR so users aren't forced to run Steam and can choose whichever runtime they want, including [Monado](https://monado.dev/).

**Comment by BastiaanOlij** [comment]
Finally fixed it properly with #49 thanks to a lot of help on the OpenXR Khronos slack channel :)
Thanks to those who helped test this.
